src/home_audit.cli.ts

```diff
@@ -21,6 +22,5 @@ if(fs.existsSync(input_string)) {
 }
 
 let input_obj: any = JSON.parse(input_string.toString())
-const result = validate(input_obj.building_unit || input_obj)
-
+const result = (input_obj.building_unit?.about || input_obj.about) ? validate(input_obj.building_unit || input_obj) : validate(translateHomeValues(input_obj))
```

Comment:
Instead of return with `validate` in the case of a flat home, we should probably use the flat response (I think `validate_home_audit`)